specification/client/chat.client.tsp

```diff
@@ -17,3 +17,5 @@ using Azure.ClientGenerator.Core;
 @@visibility(ChatCompletionStreamResponseDelta.tool_calls, Lifecycle.Read);
 
 @@clientName(Chat.createChatCompletion, "CompleteChat");
+
+@@scope(Chat.getChatCompletionMessages, "!csharp");
```

Comment:
Chris just merged a PR that adds support for this operation, so I think this can be removed.